docker-compose.yml

```diff
@@ -55,6 +62,11 @@ services:
       - USE_DEFAULT_STORAGE_CLASS=1
     volumes:
       - /etc/localtime:/etc/localtime
+      - .:/etc/yig
+      - .:/var/log/yig
+      - .:/etc/ceph

```

Comment:
why CEPH ?

Reply:
Hi rhsakarpos, ceph is needed by yig, because yig will manage one or more ceph clusters, so we need the ceph confs under that folder. this kind of settings are used by yig storage backend.